eu/intermodalics/tango_nodelet_manager/1.1.0/tango_nodelet_manager-1.1.0.pom

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>eu.intermodalics</groupId>
+  <artifactId>tango_nodelet_manager</artifactId>
+  <version>1.1.0</version>
+  <packaging>aar</packaging>
+  <dependencies>
+    <dependency>
+      <groupId>org.ros.rosjava_core</groupId>
+      <artifactId>rosjava</artifactId>
+      <version>[0.3, 0.4)</version>
```

Comment:
@adamantivm I _think_ the messages depend on versions of message_generation that are hardcoded in genjava. I'm not an expert in this particular item, but take a look at this file: https://github.com/rosjava/rosjava_bootstrap/blob/kinetic/gradle_plugins/src/main/groovy/org/ros/gradle_plugins/CatkinPlugin.groovy
If I'm not mistaken, that plugin generates a build.gradle template, which is then used to compile the messages. If you take a look at the method `generateMessageArtifact`, you will see a hardcoded dependency to message_generation[0.2, 0.3).
I offer to test if this is really what's causing this version number dependency and send a PR to rosjava_bootstrap. Apart from that, it's an issue for maintaining this repo: having a hardcoded dependency in the code doesn't sound good to me, but IDK how to fix it right away.

Another thing is the version generated for custom packages, which is 0.0.0 by default. I don't know if currently there's a way of setting it to a certain value without doing it manually.